app/build.gradle

```diff
@@ -93,10 +92,9 @@ android {
 
     productFlavors {
         development {
-            applicationId 'org.gnucash.android.devel'
-            testApplicationId 'org.gnucash.android.devel.test'
-            resValue "string", "app_name", "GnuCash-devel"
+            applicationIdSuffix '.devel'
             versionName "${versionMajor}.${versionMinor}.${versionPatch}-dev${versionBuild}_r${gitSha()}"
+            resValue "string", "app_name", "GnuCash-devel"
```

Comment:
Shouldn't it be  `GnuCash Pocket-devel` or `GnuCash Pocket (dev)` or `GnuCash Pocket (alpha)` or similar?

Reply:
It's just for development. Also "too long" names in Android don't look good.